README.md

```diff
@@ -1,150 +1,158 @@
 # Examples
+This repo contains a collection of PyTorch examples adapted for usage on the Robbie platform.
 
-This repo contains a collection of PyTorch examples adapted for usage on the Positron/Robbie platform.
-Each example will, by default, run on a Nvidia A10 GPU.
-
-Prerequisites:
-- Python 3.10 or greater installed
-- pip installed
-- You must receive an email invite from Positron/Robbie to use the service.
+Prerequisites
+- Python 3.9 or greater
+- pip
 
 For support, email: support@robbie.run
-  
-
-## Setup
-
-### Step 1. In the inviation email click on the "Join Positron" link
-
-<img width="621" alt="image" src="https://github.com/user-attachments/assets/6c351dee-8ce8-4f74-8e0e-3779d2542eec">
 
-Click on the "sign up" link and you will taken to a login screen where you will an account with a username and password.
+[Setup](#setup)  
+[Run Your First Job](#run-your-first-job)  
+[Command Runner](#command-runner)
+[Images](#images)
 
-<img width="352" alt="image" src="https://github.com/user-attachments/assets/bd782a7b-107e-4e05-ad50-496d7c127d38">
+## Setup
 
+### Step 1. Sign Up for Robbie
+Visit https://robbie.run/portal/signup and sign up with your email address or Google. 
 
-### Step 2. Create and activate a python virtual environment. 
+Check your email for a message from Robbie and click Verify Your Account in the email.
 
-Navigate to a directory that you would like the environment to live in and run the venv command.
+### Step 2. Download Robbie Examples to your local machine
+[https://github.com/Positron-Networks/robbie-examples](https://github.com/Positron-Networks/robbie-examples)
 
-On Windows or Mac enter the following at the command line/shell:
+### Step 3. Create and activate a Python virtual environment
+Navigate to a directory where you would like the environment to live and create a virtual environment.
```

Comment:
Suggest adding "This should NOT be the location that your codebase lives to avoid unnecessarily uploading virtual environment files to the Robbie Job Runner."